src/constants/relationships.js

```diff
@@ -0,0 +1,8 @@
+const RELATIONSHIPS = {
```

Comment:
Rather than `RELATIONSHIPS`, I think `RELATIONSHIP_TYPE` would probably be better. It's more targeted as well.